server/src/utils/getConfig.ts

```diff
@@ -15,33 +15,52 @@ import {
   VV_LOGO_URL_ENV_NAME
 } from '../constants';
 
-import { ServerConfigModel, ClientConfigModel } from '../models/configModel';
+import { ServerConfigModel, ClientConfigModel, PostCallSurveyType } from '../models/configModel';
 import DefaultConfig from '../defaultConfig.json';
 
+export const getDefaultConfig = (): typeof DefaultConfig => {
```

Comment:
Added this and refctored throughout this file to enable mocking and unit testing of defaultConfig.json in getConfig.test.ts